spec/mastermind/game_spec.rb

```diff
@@ -66,45 +92,50 @@ def get_user_input
 
         expect(game.over?).to be true
       end
+
+      xit "gets the next best guess forom the AI" do
+        game = Game.new(stubDisplay, stubAI)
+        game.start
+        
+        game.guess
+
+      end
     end
 
     describe "#over?" do
       it "is over when the exact matches is 4" do
-        input.inputs = [4]
-        game = Game.new(output, input)
+        stubDisplay.set_exact_matches(4)
         game.start
 
         expect(game.over?).to be false
```

Comment:
Dislike multiple assertions in this test. This actually exposes that temporal coupling and now you've cemented your implementation with a test